Source/BoundaryConditions/PML.cpp

```diff
@@ -536,10 +539,11 @@ PML::PML (const int lev, const BoxArray& grid_ba, const DistributionMapping& gri
           int do_moving_window, int /*pml_has_particles*/, int do_pml_in_domain,
           const bool do_multi_J,
           const bool do_pml_dive_cleaning, const bool do_pml_divb_cleaning,
-          int max_guard_EB,
+          int max_guard_EB, const amrex::Real v_sigma_sb,
           const amrex::IntVect do_pml_Lo, const amrex::IntVect do_pml_Hi)
     : m_dive_cleaning(do_pml_dive_cleaning),
       m_divb_cleaning(do_pml_divb_cleaning),
+      m_v_sigma_sb(v_sigma_sb),
```

Comment:
Do we really need to store `v_sigma_sb` in the `PML` object here? 
It seems that we could directly use the input argument (`v_sigma_sb`) in the body of the constructor here (see inline comments below).

Reply:
Not that, if the above is indeed correct, then you could remove `m_v_sigma_sb` from `PML.H`